.travis.yml

```diff
@@ -1,48 +1,81 @@
 # Travis Continuous Integration Configuration File For ROS Control Projects
-# Author: Dave Coleman
-language:
-  - cpp
-  - python
-python:
-  - "2.7"
+# Author: Dave Coleman, Enrique Fernandez
+sudo: required
+dist: xenial
```

Comment:
Are you sure if Ubuntu 16.04 is already available on Travis CI?

From [one of the logs](https://s3.amazonaws.com/archive.travis-ci.org/jobs/133222368/log.txt), it seems the job is running on 12.04 to me:
```
Build dist: xenial
[34m[1mBuild image provisioning date and time[0m
Tue Dec 15 16:36:18 UTC 2015
[34m[1mOperating System Details[0m
Distributor ID:	Ubuntu
Description:	Ubuntu 12.04.5 LTS
Release:	12.04
Codename:	precise
```